File: jambeth/jambeth-service/src/main/java/de/osthus/ambeth/metadata/RelationMember.java
package de.osthus.ambeth.metadata;

import de.osthus.ambeth.annotation.CascadeLoadMode;

public abstract class RelationMember extends Member
{
	@Override
	public Object getNullEquivalentValue()
	{
		return null;
	}

	public abstract CascadeLoadMode getCascadeLoadMode();

	public abstract boolean isManyTo();
}

File: ambeth/Ambeth.Service/ambeth/metadata/RelationMember.cs
using De.Osthus.Ambeth.Annotation;
using De.Osthus.Ambeth.Typeinfo;
using System;

namespace De.Osthus.Ambeth.Metadata
{
    public abstract class RelationMember : Member
    {
        public override Object NullEquivalentValue
        {
            get
            {
                return null;
            }
        }

        public abstract CascadeLoadMode CascadeLoadMode { get; }

        public abstract bool IsManyTo { get; }
    }
}